src/devices/Ads1115/Ads1115.cs

```diff
@@ -285,7 +274,7 @@ private void DisableAlertReadyPin()
                 (byte)Register.ADC_CONFIG_REG_HI_THRESH, 0x7F, 0xFF
             };
             _i2cDevice.Write(writeBuff);
-            if (_gpioController != null)
+            if (_gpioController is object)
```

Comment:
at the risk of  starting a war, I think `is not null` is more clear than `is object`. They generate the same code ([`is not null`](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDYB8ACAmARgFgAoHAZgAJ9KBhSgb1MpeqpwBZKBZAChwIAGSlACUjZqyloAZpV5RKaRQDsA9jEoqArhgyimJKcdYCAnAtEBuSSYC+tyg5J2gA=) vs [`is object`](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDYB8ACAmARgFgAoHAZgAJ9KBhSgb1MpeqpwBZKBZAChwIAGSlACUjZqyloAZpV5RKaRQHtgAKzgBjGKKYkph1gICcC0QG5JRgL7XKdkjaA)).

Adding @jaredpar to make sure dissenting opinions way in.

Reply:
I sorta figured it isn't using `not null` due to language version requirements for the repo. 
But in general re `is object` vs `is not null`, this feels like something that is going to keep coming up as people are opinionated on this; there's no technical advantage either way, it's purely preference driven.
So tl;dr, 🍿 (Hi Jared :)

Reply:
I'm guessing that people are going to have to get used to seeing it both ways. People that are super in the weeds of C# like the language team might decide that `is object` is extremely clear to them and choose it in general. Other people working on their company LOB codebases might decide on `is not null` because it seems easier for Joe/Jane Developer to understand what's going on. Each team/codebase will choose what they feel is clear and understandable to them (and readers of the code), though what that means will vary in context.

Reply:
no reason to new set the lang version explicitly imho:

```xml
<PropertyGroup>
   <LangVersion>8.0</LangVersion>
</PropertyGroup>
```